__tests__/app.test.js

```diff
@@ -121,4 +121,64 @@ describe('GET /api/articles/:article_id', () => {
             expect(msg).toBe('path not found')
         })
     })
+})
+
+describe('GET /api/articles/:article_id/comments', () => {
```

Comment:
testing looks good but I do think you're missing a test case:
What happens if we ask for the comments for an article that exists but has no comments, will you get a 200 and an empty array?

Reply:
done, thanks